server/routerlicious/packages/services-utils/src/apiCounters.ts

```diff
@@ -0,0 +1,72 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export interface IApiCounters {
```

Comment:
@kekachmar Is this the same ApiCounter implementation we have in the FRS repo? 😄

Reply:
Yes, we decided to move this over to the open-source repo

Reply:
Got it. When I implemented that in FRS repo, the only consumer was the AzureBlobFS... Now, we should make AzureBlobFS and any other consumers since then use the version from OSS as well. Since you were the one who moved this logic here, can you create a work item for that just so that we don't forget about it? Thanks!